files/czertainly.yml

```diff
@@ -3,6 +3,26 @@
   hosts: all
   connection: local
 
+  pre_tasks:
```

Comment:
Isn't it done automatically?

https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable